discovery-provider/plugins/notifications/src/email/notifications/index.ts

```diff
@@ -301,6 +304,11 @@ export async function processEmailNotifications(
     }
     const startOffset = now.clone().subtract(days, 'days')
     const users = await getUsersCanNotify(identityDb, frequency, startOffset)
```

Comment:
oh we should still check if it shouldSendEmail with the notification settings

Reply:
ooo yes good point. maybe some point down the line we can consolidate getUsersCanNotify into userNotificationSettings since it seems to be doing some similar things but def not necessary for now